src/main/java/by/dragonsurvivalteam/dragonsurvival/client/render/entity/dragon/DragonArmorRenderLayer.java

```diff
@@ -249,13 +283,10 @@ public static String itemToResLoc(Item item) {
 		if (item == Items.AIR) return null;
 
 		ResourceLocation registryName = ResourceHelper.getKey(item);
-		if (registryName != null) {
-			String[] reg = registryName.toString().split(":");
-			String loc = reg[0] + "/" + reg[1] + ".png";
-			return stripInvalidPathChars(loc);
-		}
-		return null;
-	}
+        String[] reg = registryName.toString().split(":");
+        String loc = reg[0] + "/" + reg[1] + ".png";
+        return stripInvalidPathChars(loc);
+    }
```

Comment:
why not use `getNamespace` and `getPath`

would also recommend just dropping the `stripInvalidPathChars` since a resource location should not contain these things and it's just unneeded processing

Reply:
⭐